### DISCUSSION TITLE
Unable to Scan Inverted QR Code

### DISCUSSION THREAD

**Discussion by AlfredAfutu** [journeyapps/zxing-android-embedded#668] [Q&A]
I am unable to scan the QR code attached using version 4.1.0 on a Google Pixel 3

It looked like an inverted code so i tried setting the SCAN_TYPE to 1 but to no avail.

The content of the code is in the format URL:https://domain_name/alphanumeric_characters

I would be glad if i can be put on the right direction to solve the issue.

Thanks in advance
<img width="128" alt="Screenshot 2021-04-08 at 16 47 12" src="https://user-images.githubusercontent.com/5557324/114075468-dd043b80-98a5-11eb-913d-bab63368c8aa.png">

**Comment by saba7afi** [comment]
any update please? I've the same issue.

**Comment by rkistner** [comment]
The image above works for me using the sample app, using either `Intents.Scan.INVERTED_SCAN` or `Intents.Scan.MIXED_SCAN` for SCAN_TYPE:

https://github.com/journeyapps/zxing-android-embedded/blob/aa129af542f0b09c21dcd1731b678a07c9ff9a78/sample/src/main/java/example/zxing/MainActivity.java#L52-L62
